test/commands/user/create.nut.ts

```diff
@@ -63,13 +63,14 @@ describe('creates a user from a file and verifies', () => {
         ensureExitCode: 0,
       }
     ).jsonOutput;
-    // expect(output.jsonOutput).to.have.property('result').with.all.keys(['orgId', 'permissionSetAssignments', 'fields']);
     expect(output.result).to.have.all.keys(['orgId', 'permissionSetAssignments', 'fields']);
     expect(output.result.permissionSetAssignments).to.deep.equal(['VolunteeringApp']);
     createdUserId = output.result.fields.id as string;
   });
-
-  it('verifies the permission set assignment in the org', async () => {
```

Comment:
I don't love this, but was unable to find another way to verify that the PS was actually assigned. It might require bumping `testkit` to core3